init-files/main.typ

```diff
@@ -1,4 +1,4 @@
-#import "./tongjithesis.typ": *
+#import "@preview/paddling-tongji-thesis:0.1.1": *
```

Comment:
This may not be very fitting to be in main branch? Considering users may clone this repositry and modify `tongjithesis.typ`, but as this is imported from the package, the behavior and results of the actual compilation may be confusing.

I suggest to maintain this change in a new branch, or as a git patch which can be easily applied to the repository with `git apply` command when we need to make a release to the typst package registry.

Reply:
Please accept the invitation and you can do anything in this repo now